### ISSUE TITLE
made the classes needed to make this project work 

### ISSUE THREAD

**Issue by RoelofWobben** [RoelofWobben/GroupProject#1]
I made the classes and I hope you both are happy with it.
If not so, respons please.

**Comment by Aurora2500** [comment]
I don't see any of them, did you push any changes?

**Comment by RoelofWobben** [comment]
you are right
I thought I have pushed them

**Comment by RoelofWobben** [comment]
oke, pushed it again and I see them now. 
Sorry , maybe it had to do with the name change

**Comment by Aurora2500** [comment]
Yeah they're visible now, Now what's left I think are some classes to manage how the user would interact with it and how it would write to file and so on.

**Comment by RoelofWobben** [comment]
hmm.  on which classes do you think.  A PrintToFileClass ?

**Comment by Aurora2500** [comment]
I'd say so, and the Program class can be used for user interaction.
